VideoAPI/VideoApi.Domain/RoomParticipant.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using VideoApi.Domain.Ddd;
+
+namespace VideoApi.Domain
+{
+    public class RoomParticipant : Entity<long>
+    {
+        public RoomParticipant(long roomId, Guid participantId, DateTime enterTime)
+        {
+            RoomId = roomId;
+            ParticipantId = participantId;
+            EnterTime = enterTime;
```

Comment:
I don't think the Enter time should be passed as a parameter, rather set as the current time when the object is initialised

Reply:
done